code/controllers/configuration.dm

```diff
@@ -59,6 +59,7 @@
 	var/jobs_have_minimal_access = 0	//determines whether jobs use minimal access or expanded access.
 	var/jobs_have_maint_access = 0 		//Who gets maint access?  See defines above.
 	var/sec_start_brig = 0				//makes sec start in brig or dept sec posts
+	var/gamemode_disable_jobs = 0 		//Disables job if the gamemode supports

```

Comment:
why not use the disabled jobs list instead? if it's null it doesn't disable jobs, if it's got items, it's enabled.